go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	github.com/joho/godotenv v1.5.1
 	github.com/jpillora/longestcommon v0.0.0-20161227235612-adb9d91ee629
 	github.com/keboola/go-client v1.26.4
-	github.com/keboola/go-utils v0.10.1
+	github.com/keboola/go-utils v0.10.2-0.20240521102004-ad17a99677bb
```

Comment:
Utils that supports new environment variable that loads projects from `projects.json` file.